package.json

```diff
@@ -3,12 +3,13 @@
   "scripts": {
     "build": "pulp build --to dist/app.js",
     "watch": "pulp -w build --to dist/app.js",
-    "watch-fast": "webpack --entry ./entry.js --output-filename dist/app.js --progress --watch"
+    "watch-fast": "webpack --mode=development --entry ./entry.js --output-filename app.js --progress --watch"
```

Comment:
For some reason it was still producing the file in ./dist. I added --output-path ./dist just in case.
I also added package-lock.json to .gitignore by the way.